Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
-        // https://github.com/sonata-project/SonataAdminBundle/issues/151
-        //if it is a standard browser use preventDefault otherwise it is IE then return false
-        if(event.preventDefault) {
+    "stopEvent" (event) {
+
+        /*
+         * https://github.com/sonata-project/SonataAdminBundle/issues/151
+         *if it is a standard browser use preventDefault otherwise it is IE then return false
+         */
+        if (event.preventDefault) {
+
             event.preventDefault();
+
         } else {
+
             event.returnValue = false;
+
         }
 
-        //if it is a standard browser get target otherwise it is IE then adapt syntax and get target
-        if (typeof event.target != 'undefined') {
+        // If it is a standard browser get target otherwise it is IE then adapt syntax and get target
+        if (typeof event.target !== "undefined") {
+
             targetElement = event.target;
+
         } else {
+
             targetElement = event.srcElement;
+
         }
 
         return targetElement;
+
     },
 
-    add_filters: function(subject) {
-        Admin.log('[core|add_filters] configure filters on', subject);
-        jQuery('a.sonata-toggle-filter', subject).on('click', function(e) {
+    "add_filters" (subject) {
+
+        Admin.log("[core|add_filters] configure filters on", subject);
+        jQuery("a.sonata-toggle-filter", subject).on("click", (e) => {
+
             e.preventDefault();
             e.stopPropagation();
 
-            if (jQuery(e.target).attr('sonata-filter') == 'false') {
+            if (jQuery(e.target).attr("sonata-filter") == "false") {
+
                 return;
+
             }
 
-            Admin.log('[core|add_filters] handle filter container: ', jQuery(e.target).attr('filter-container'))
+            Admin.log("[core|add_filters] handle filter container: ", jQuery(e.target).attr("filter-container"));
+
+            const filters_container = jQuery(`#${jQuery(e.currentTarget).attr("filter-container")}`);
 
-            var filters_container = jQuery('#' + jQuery(e.currentTarget).attr('filter-container'));
+            if (jQuery("div[sonata-filter=\"true\"]:visible", filters_container).length == 0) {
 
-            if (jQuery('div[sonata-filter="true"]:visible', filters_container).length == 0) {
                 jQuery(filters_container).slideDown();
-            }
 
-            var targetSelector = jQuery(e.currentTarget).attr('filter-target'),
-                target = jQuery('div[id="' + targetSelector + '"]', filters_container),
-                filterToggler = jQuery('i', '.sonata-toggle-filter[filter-target="' + targetSelector + '"]')
-            ;
+            }
 
+            let targetSelector = jQuery(e.currentTarget).attr("filter-target"),
+                target = jQuery(`div[id="${targetSelector}"]`, filters_container),
+                filterToggler = jQuery("i", `.sonata-toggle-filter[filter-target="${targetSelector}"]`);
```

Comment:
- eslint: Line 317 exceeds the maximum line length of 80.
- eslint: 'filterToggler' is never reassigned. Use 'const' instead.
- eslint: Variables within the same declaration block should be sorted alphabetically.